cmd/controller/collectdiskinfo_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2018 OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	apis "github.com/openebs/node-disk-manager/pkg/apis/ndm/v1alpha1"
+	"github.com/openebs/node-disk-manager/pkg/udev"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCollectDiskInfo(t *testing.T) {
+	newUdev := udev.NewUdev()
+	if newUdev == nil {
+		t.Error("Unable to create new udev object.")
+	}
+	defer newUdev.UnrefUdev()
+	disks, err := exec.Command("lsblk", "-r", "-d", "-n", "-o", "type,name,maj:min").Output()
```

Comment:
Plz discuss with the team if making use of actual binary is okay?

Reply:
cc: @pawanpraka1 we have decided to use it in unit test.

Reply:
@gkGaneshR is right. We should not use any binaries while UT.